e2e/deployers/disapp.go

```diff
@@ -75,14 +100,33 @@ func (d DiscoveredApp) Deploy(ctx types.TestContext) error {
 		return err
 	}
 
-	log.Info("Workload deployed")
-
 	return nil
 }
 
 // Undeploy deletes the workload from the managed clusters.
 func (d DiscoveredApp) Undeploy(ctx types.TestContext) error {
 	log := ctx.Logger()
```

Comment:
Same issue as deploy, keep the existing function and enhance it with recipe stuff. If the function is too big and contain too many details refactor out the details.